docs/samples/README.md

```diff
@@ -147,10 +147,21 @@ $ git checkout gh-pages
 $ cd docs/samples
 ```
 
+1. Ensure that the [samples top level pom.xml](pom.xml) has the
```

Comment:
Do not tell them to edit a POM file. If anything tell them to use the coherence.version property when building